gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -1070,6 +1072,30 @@ gst_tensor_src_iio_set_property (GObject * object, guint prop_id,
         self->channels_enabled = CHANNELS_ENABLED_ALL;
       } else if (!g_strcmp0 (param, CHANNELS_ENABLED_AUTO_CHAR)) {
         self->channels_enabled = CHANNELS_ENABLED_AUTO;
+      } else {
```

Comment:
This is not related with this PR, so I wouldn't apply it to this PR.
However, for property decision, please use case insensitive mechanisms instead of g_strcmp0. Glib has case insensitive strcmp API.

Reply:
Noted, will upload a new PR for case insensitive comparison.

Reply:
Added #1360 .